app/src/main/java/com/example/marvelapp/presentation/MainActivity.kt

```diff
@@ -20,21 +21,27 @@ class MainActivity : AppCompatActivity(), ItemClicked {
     private lateinit var binding: ActivityMainBinding
 
     override fun onCreate(savedInstanceState: Bundle?) {
-
         super.onCreate(savedInstanceState)
-
         binding = ActivityMainBinding.inflate(layoutInflater)
         setContentView(binding.root)
+        floatingButton()
+
         presenter = MarvelAppPresenter(
             MarvelAppModel(
                 GetCharactersUseCaseImpl(MarvelServiceImpl()),
-                SaveCharactersDataBaseUseCaseImpl(LocalMarvelDatabaseImpl(CharacterRealmMapper))
+                SaveCharactersDataBaseUseCaseImpl(LocalMarvelDatabaseImpl(CharacterRealmMapper)),
+                LoadCharactersUseCaseImpl(LocalMarvelDatabaseImpl(CharacterRealmMapper))
             ),
             MarvelAppView(this, binding)
         )
         presenter.fetchHeroes()
     }
 
+    private fun floatingButton() {
```

Comment:
please put here a more representative name, this name doesn't make sense

Reply:
Done thx